conf.py

```diff
@@ -538,6 +550,15 @@ def run(self):
 # .. xapianrunexample:: search_filters2
 directives.register_directive('xapianrunexample', XapianRunExample)
 
+class XapianTrain(LiteralInclude):
+    option_spec = { }
+
+    def run(self):
+        os.system("xapian-train --db=db data/training_data.txt ListNET_Ranker")
+        return super(XapianTrain, self).run()
+
+
+directives.register_directive('xapiantrain', XapianTrain)
```

Comment:
Extra blank line before this one.

Also, for the other directives we have a "Usage:" comment.